substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SystemPropertiesSupport.java

```diff
@@ -65,7 +66,36 @@
                     "java.specification.version",
                     "java.vm.specification.name",
                     "java.vm.specification.vendor",
-                    "java.vm.specification.version"
+                    "java.vm.specification.version",
+                    "sun.arch.data.model"
+    };
+
+    /**
+     * System properties are unsupported by default.
+     */
+    private static final String[] UNSUPPORTED_PROPERTIES = {
```

Comment:
"unsupported" is in general the wrong term for all of this PR. They are of course supported (since you can manually set them), they just are not set currently by default.

Reply:
fixed by changing to INCOMPATIBLE_PROPERTIES